sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -104,15 +96,19 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
     mapboxRenderer: MapboxRenderer,
     mapboxWidgetRenderer: MapboxWidgetRenderer,
     translucentSurface: Boolean,
-    antialiasingSampleCount: Int,
+    antialiasingSampleCount: Int
   ) {
     this.translucentSurface = translucentSurface
     this.mapboxRenderer = mapboxRenderer
     this.widgetRenderer = mapboxWidgetRenderer
     this.eglCore = EGLCore(translucentSurface, antialiasingSampleCount)
+    this.fpsManager = FpsManager()
```

Comment:
Nit : add Handler as a parameter and init after `RenderHandlerThread().apply { start()`

Reply:
this will actually not init `fpsManager` as an object in ctor, see https://github.com/mapbox/mapbox-maps-android/pull/1477#discussion_r928672237